ansible-roles/ansible-role-springboot_foundation/tasks/lvm.yml

```diff
@@ -0,0 +1,76 @@
+---
+
+- name: "Create LV for /opt/springboot"
+  lvol:
+    vg: "{{ sb_dedicated_vg_opt }}"
+    lv: lv_opt_springboot
+    size: 1g
+    state: present
+  become: "{{ sb_become }}"
+  when: sb_dedicated_lv_opt
+
+- name: "Initialise the XFS filesystem for /opt/springboot"
+  filesystem:
+    fstype: xfs
```

Comment:
fs type and sizes might become parameter in a future enhancement.

Reply:
Surely will, later ;-)